aips/create_dip.py

```diff
@@ -27,6 +27,11 @@
 THIS_DIR = os.path.abspath(os.path.dirname(__file__))
 LOGGER = logging.getLogger("create_dip")
 
+PREMIS_VERSIONS_NS = {
+    "2.2": "info:lc/xmlns/premis-v2",
+    "3.0": "http://www.loc.gov/premis/v3",
+}
```

Comment:
You could use `metsrw.plugins.premisrw.utils.*` instead.

Reply:
It was a little ugly to use `metsrw.plugins.premisrw.utils.`PREMIS_VERSIONS_MAP[premis_version]["namespaces"]["premis"]` to update the `metsrw.utils..NAMESPACES` copy, but I'll give it another look.

Reply:
Done, thanks!